CHANGES.txt

```diff
@@ -6,6 +6,13 @@ Unreleased
 ==========
 
 
+2022/07/04 0.27.1
+=================
+
+- Fixed regression introduced by ``0.27.0`` resulting in unavailable servers if all
```

Comment:
I think this exceeds the max 80 char line length, seems like GH did not submit my changed suggestion :( Sorry.